sdk/simulator-service/src/implementation/utils/SimulatorUtils.ts

```diff
@@ -1,4 +1,4 @@
-import type {
+  import type {
```

Comment:
I see some formatting looks a bit off, I'm wondering if you are using an editor that formats according to lint rules